test/scheduled_skills.py

```diff
@@ -0,0 +1,29 @@
+from datetime import datetime, timedelta
+import unittest
+
+from mycroft.skills.scheduled_skills import ScheduledSkill
+from mycroft.util.log import getLogger
+
+__author__ = 'eward'
+
+logger = getLogger(__name__)
+
+
+class ScheduledSkillTest(unittest.TestCase):
```

Comment:
This file should be inside the /test/skill/scheduled_skills.py instead of test/scheduled_skills.py